packages/plugin-core/src/components/lookup/LookupControllerV3.ts

```diff
@@ -230,96 +235,507 @@ export class LookupControllerV3 implements ILookupControllerV3 {
     return quickpick;
   }
 
-  async show(
-    opts: CreateQuickPickOpts & {
-      /**
-       * Don't show quickpick
-       */
-      nonInteractive?: boolean;
-      /**
-       * Initial value for quickpick
-       */
-      initialValue?: string;
-      provider: ILookupProviderV3;
-    }
-  ): Promise<DendronQuickPickerV2> {
-    const { quickpick } = await this.prepareQuickPick(opts);
-    return this.showQuickPick({ ...opts, quickpick });
-  }
-
-  onHide() {
+  public onHide() {
     const ctx = "LookupControllerV3:onHide";
-    this._quickpick?.dispose();
-    this._quickpick = undefined;
+    this._quickPick?.dispose();
+    this._quickPick = undefined;
     this._cancelTokenSource?.dispose();
-    this._view?.deregisterController();
+
+    this._disposables.forEach((disposable) => disposable.dispose());
```

Comment:
If we do this, does the views dispose themselves as well?

Reply:
It should - both the views implement Disposable, and they get added to this `_disposables` array:

```typescript
      this._disposables.push(new LookupPanelView(this._viewModel));
      ...
      this._disposables.push(
      new LookupV3QuickPickView(quickpick, this._viewModel, this._provider.id)
    );
 ```